cypress/e2e/hub/remote-registries.cy.ts

```diff
@@ -16,34 +14,23 @@ describe('Remote Registry', () => {
     cy.hubLogin();
   });
 
-  after(() => {
```

Comment:
This change was added to make sure of clean up being successfully done. Leveraging the test signature to delete the resources created by a specific test run. Lets not remove this.

Reply:
Also, this change does not seem to be related to the EE change. Please, add separate PRs to different concerns.